app/views/events/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Events</h1>
+<% if current_user && current_user.admin %>
+  <%= link_to 'New Event', new_event_path, class: "btn btn-secondary" %>
+<% end %>
+<% @events.each do |event| %>
+  <div class="col-sm-6">
+    <div class="card" style="width: 18rem;">
```

Comment:
Don't use inline styles. Use something like `w-25`. If that is too big, we can generate width helpers dynamically.